app/templates/settings/settings-general.hbs

```diff
@@ -147,6 +147,12 @@
         <label for="settings__general-lock-on-copy">{{res 'setGenLockCopy'}}</label>
     </div>
 
+    <div>
```

Comment:
This must not be visible on web. Please wrap with `{{#if canAutoType}}` and use `AutoType.enabled` for it.